zilsd.adoc

```diff
@@ -1,18 +1,37 @@
-[[chapter2]]
-== Load/Store pair instructions (Zilsd)
 
-The Zilsd extension provides load/store pair instructions for RV32 analogous to the existing RV64 doubleword load/store instructions. The existing RV64 encodings are reused, which are reserved in RV32 without this extension.
+== Load/Store pair
+
+The Zilsd & Zcmlsd extensions provide load/store pair instructions for RV32 analogous to the existing RV64 doubleword load/store instructions. The existing RV64 encodings are reused, which are reserved in RV32 without this extension.
```

Comment:
not true - the 32-bit encodings re reserved for RV32, but the 16-bit encodings replace Zcf

Reply:
right. need to get this correct without overcomplicating it.

Reply:
fixed in 04063d5